evaporate.js

```diff
@@ -1273,12 +1273,14 @@
 
         function uploadKey(fileUpload) {
             // The key tries to give a signature to a file in the absence of its path.
-            // "<filename>-<mimetype>-<modifieddate>-<filesize>"
+            // Need to include path (if present) in case the same file is present in multiple paths/folders
+            // "<filename>-<mimetype>-<modifieddate>-<filesize>-<path>"
```

Comment:
For security reasons, the browsers does not allow access to absolute path & file systems directly to Javascript. Some specific browsers reportedly do allow this when a user sets security settings on their instance but reports are that even with these security workarounds, the solution is not reliable.

I don't think it is appropriate to build this type of browser polyfill thingy to get the file path, particularly given the security issues involved.